wordpress-mobile/WordPress-iOS
# Feature/8622 search ad attribution implementation

Fixes #8622 

This PR implements the Search Ad Attribution API and send the given data to Tracks as an event.

To test:

- Run the project on a device.
- Check if the new event arrives to Tracks (`wpios_searchads_attribution_detail_received`)
- It should arrive just once, the first time you run the app.
